src/App.js

```diff
@@ -1,11 +1,53 @@
+import { Component } from 'react';
+import { Route, Switch } from 'react-router-dom';
+import axios from 'axios';
+
 import './App.css';
+import MoviesDscription from './components/moviesDescription';
+import Movies from './components/movies';
+import MoviesReviews from './components/moviesReviews';
+
+class App extends Component {
+    constructor(props) {
+      super(props);
+   
+      this.state = {
+        movies: [],
+      };
+    }
+
+    componentDidMount(){
+      axios.get(`
+      https://api.themoviedb.org/3/movie/popular?api_key=ed4f70d18fa8cbc03fb59ce04847e1bc`)
```

Comment:
same API key is being used in multiple components, store it in a different file as constant and import where required